index.html

```diff
@@ -0,0 +1,699 @@
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="latest-deal-img-5 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              Leadership & Management Course
+            </h3>
+            <ul class="flex price">
+              <li class="cross-text">$600</li>
+              <li class="main-price">$29.99</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="latest-deal-img-6 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              Ultimate Photography Course
+            </h3>
+            <ul class="flex price">
+              <li class="cross-text">$50</li>
+              <li class="main-price">$39.99</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="latest-deal-img-7 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              CES goes fully digital in 2021
+            </h3>
+            <ul class="flex price">
+              <li class="cross-text">$90</li>
+              <li class="main-price">$49.99</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="latest-deal-img-8 latest-news-image"></div>
+          <div class="latest-news-text">
+            <h3>
+              The Complete Technical SEO Course
+            </h3>
+            <ul class="flex price">
+              <li class="cross-text">$120</li>
+              <li class="main-price">$69.99</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer id="bottom-footer">
+      <div class="top-footer">
+        <ul class="flex social-icons">
+          <li class="social-icon">
+            <i class="fab fa-facebook-square"></i>
+          </li>
+          <li class="social-icon">
+            <i class="fab fa-twitter"></i>
+          </li>
+          <li class="social-icon">
+            <i class="fab fa-youtube"></i>
+          </li>
+          <li class="social-icon">
+            <i class="fab fa-instagram"></i>
+          </li>
+          <li class="social-icon">
+            <i class="fab fa-pinterest-p"></i>
+          </li>
+          <li class="social-icon">
+            <i class="fas fa-rss"></i>
+          </li>
+          <li class="social-icon">
+            <i class="fas fa-envelope"></i>
+          </li>
+        </ul>
+        <ul class="flex links-list">
+          <li class="link-item">EVENTS</li>
+          <li class="link-item">ABOUT</li>
+          <li class="link-item">ADVERTISE</li>
+          <li class="link-item">JOBS</li>
+          <li class="link-item">CONTACT</li>
+        </ul>
+      </div>
+      <div class="down-footer">
+        <div class="flex copyright-text">
+          <span
+            >© 2006–2020 The Next Web B.V. Made with ♥ in Amsterdam. Powered
+            by</span
+          >
+        </div>
+      </div>
```

Comment:
**Footer**

- [ ] It contains the TNW logo with a copyright paragraph ([link to page](https://thenextweb.com/)) on the original page, but on your page the logo is missing. Please kindly implement it.

![image](https://user-images.githubusercontent.com/60184961/91214051-36569a80-e71b-11ea-8007-860884ae3e8a.png)